modules/rosa-configure.adoc

```diff
@@ -0,0 +1,271 @@
+:system-module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+[id="rosa-configure_{context}"]
+= Configuring `rosa`
+
+[role="_abstract"]
+Use the following commands to configure the `rosa` CLI.
+
+[id="rosa-login_{context}"]
+== login
+
+Logs in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token by from link:https://cloud.redhat.com/openshift/token/rosa[the {product-title} token page].
+
+`rosa` looks for a token in the following priority order:
+
+* Command-line arguments
+* The `ROSA_TOKEN` environment variable
+* The `rosa` configuration file
+* Interactively from a command-line prompt
+
+.Syntax
+[source,terminal]
+----
+$ rosa login [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--client-id
```

Comment:
I prefer it this way, rather than after the argument name because it catches the users eye better. But in the next release, I may make changes. Due to severe time constraints, this needs to just be a best effort.